### ISSUE TITLE
Error when delete tags the cache before caching itself

### ISSUE THREAD

**Issue by DimaKalbfleysh** [Krukov/cashews#135]
If you call the delete_tags function at the start of the script/server, the error falls:
`AttributeError: 'Redis' object has no attribute '_client'`

For example as in the code below, slightly modified this example https://github.com/Krukov/cashews/blob/8fb81ca97bb548587fd59cb657f06e1664751189/examples/invalidation_by_tags.py

```python
import asyncio
import random
import typing as t

from cashews import cache

redis_url = "redis://"
cache.setup(redis_url)


@cache(ttl="1h", tags=["items", "user_data:{user_id}"])
async def get_items(user_id: int, some_id: t.Optional[int] = None):
    return [f"{user_id}_{random.randint(1, 10)}" for i in range(10)]


FIRST_USER = 1
SECOND_USER = 2


async def main():
    # delete tags before caching
    await cache.delete_tags(f"user_data:{FIRST_USER}")  # AttributeError: 'Redis' object has no attribute '_client'

    first_user_items = await get_items(FIRST_USER)
    second_user_items = await get_items(SECOND_USER)

    assert await get_items(FIRST_USER) == first_user_items
    assert await get_items(SECOND_USER) == second_user_items

    await cache.delete_tags(f"user_data:{FIRST_USER}")
    assert await get_items(FIRST_USER) != first_user_items
    assert await get_items(SECOND_USER) == second_user_items


if __name__ == "__main__":
    asyncio.run(main())

```

Traceback 
```
Traceback (most recent call last):
  File "", line 36, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "", line 22, in main
    await cache.delete_tags(f"user_data:{FIRST_USER}")  # AttributeError: 'Redis' object has no attribute '_client'
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/cashews/wrapper/tags.py", line 87, in delete_tags
    await self._delete_tag(tag)
  File "/opt/homebrew/lib/python3.11/site-packages/cashews/wrapper/tags.py", line 91, in _delete_tag
    keys = await self.tags_backend.set_pop(self._tags_key_prefix + tag, 100)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/cashews/backends/redis/backend.py", line 265, in set_pop
    return [value.decode() for value in await self._client.spop(key, count)]
                                              ^^^^^^^^^^^^
AttributeError: 'Redis' object has no attribute '_client'
```

**Comment by Krukov** [comment]
Thanks for detailed report.  
Fix is on the way.

**Comment by Krukov** [comment]
Fixed 6.0.2 

Please reopen the issue if didn't help
